.changelogs/remove-deprecated-items-2.yml

```diff
@@ -0,0 +1,17 @@
+significance: major
+type: removed
+entry: |
+  + Removed deprecated `llms_format_decimal()` function and related unit test.
```

Comment:
Unit test information is excluded from changelogs deployed to end users. They're really developer tools that only matter, primarily, for us internally. We rarely include information about tests in the public-facing changelog.